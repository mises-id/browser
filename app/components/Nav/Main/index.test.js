import React from 'react';
import {shallow} from 'enzyme';
import {NavigationContainer} from '@react-navigation/native';

describe('Main', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<NavigationContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
