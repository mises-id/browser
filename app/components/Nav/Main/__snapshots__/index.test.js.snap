// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should render correctly 1`] = `
<ContextProvider
  value="light"
>
  <ContextProvider
    value={
      Object {
        "_childrenNavigation": Object {},
        "actions": Object {
          "dismiss": [Function],
          "pop": [Function],
          "popToTop": [Function],
          "push": [Function],
          "replace": [Function],
          "reset": [Function],
        },
        "addListener": [Function],
        "dangerouslyGetParent": [Function],
        "dismiss": [Function],
        "dispatch": [Function],
        "getChildNavigation": [Function],
        "getScreenProps": [Function],
        "goBack": [Function],
        "isFirstRouteInParent": [Function],
        "isFocused": [Function],
        "navigate": [Function],
        "pop": [Function],
        "popToTop": [Function],
        "push": [Function],
        "replace": [Function],
        "reset": [Function],
        "router": Object {
          "childRouters": Object {
            "AddBookmarkView": null,
            "BrowserView": null,
            "Create": null,
            "CreateStep2": null,
            "GeneralSettings": null,
            "Password": null,
            "Restore": null,
            "SettingsView": null,
            "Webview": null,
          },
          "getActionCreators": [Function],
          "getActionForPathAndParams": [Function],
          "getComponentForRouteName": [Function],
          "getComponentForState": [Function],
          "getPathAndParamsForState": [Function],
          "getScreenOptions": [Function],
          "getStateForAction": [Function],
        },
        "setParams": [Function],
        "state": Object {
          "index": 0,
          "isTransitioning": false,
          "key": "StackRouterRoot",
          "routes": Array [
            Object {
              "key": "id-123-0",
              "routeName": "Password",
            },
          ],
        },
      }
    }
  >
    <Connect(Main)
      navigation={
        Object {
          "_childrenNavigation": Object {},
          "actions": Object {
            "dismiss": [Function],
            "pop": [Function],
            "popToTop": [Function],
            "push": [Function],
            "replace": [Function],
            "reset": [Function],
          },
          "addListener": [Function],
          "dangerouslyGetParent": [Function],
          "dismiss": [Function],
          "dispatch": [Function],
          "getChildNavigation": [Function],
          "getScreenProps": [Function],
          "goBack": [Function],
          "isFirstRouteInParent": [Function],
          "isFocused": [Function],
          "navigate": [Function],
          "pop": [Function],
          "popToTop": [Function],
          "push": [Function],
          "replace": [Function],
          "reset": [Function],
          "router": Object {
            "childRouters": Object {
              "AddBookmarkView": null,
              "BrowserView": null,
              "Create": null,
              "CreateStep2": null,
              "GeneralSettings": null,
              "Password": null,
              "Restore": null,
              "SettingsView": null,
              "Webview": null,
            },
            "getActionCreators": [Function],
            "getActionForPathAndParams": [Function],
            "getComponentForRouteName": [Function],
            "getComponentForState": [Function],
            "getPathAndParamsForState": [Function],
            "getScreenOptions": [Function],
            "getStateForAction": [Function],
          },
          "setParams": [Function],
          "state": Object {
            "index": 0,
            "isTransitioning": false,
            "key": "StackRouterRoot",
            "routes": Array [
              Object {
                "key": "id-123-0",
                "routeName": "Password",
              },
            ],
          },
        }
      }
      theme="light"
    />
  </ContextProvider>
</ContextProvider>
`;
